frontend-instructor/src/components/login/Login.jsx

```
@@ -0,0 +1,85 @@
+            <label className="block text-sm font-medium">Email</label>
+            <input
+              type="email"
+              id="email"
+              className="w-full p-2 border rounded-md"
+              onChange={loginFormik.handleChange}
+              value={loginFormik.values.email}
+            />
+          </div>
+
+          <div>
+            <label className="block text-sm font-medium">Password</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)